src/components/services/DockerSocketProxy.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="notifs">
+        <strong v-if="running > 0" class="notif running" title="Running Containers">
+          {{ running }}
+        </strong>
+        <strong v-if="stopped > 0" class="notif stopped" title="Stopped Containers">
+          {{ stopped }}
+        </strong>
+        <strong v-if="errors > 0" class="notif errors" title="Error">
+          {{ errors }}
+        </strong>
+        <strong v-if="serverError" class="notif errors" title="Connection error to Docker Socket Proxy API">
+          ?
+        </strong>
+      </div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+export default {
+  name: "DockerSocketProxy",
+  components: {
+    Generic,
+  },
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  data: () => {
+    return {
+      running: null,
+      stopped: null,
+      errors: null,
+      serverError: false,
+    };
+  },
+  created: function () {
+    const checkInterval = parseInt(this.item.checkInterval, 10) || 0;
+    if (checkInterval > 0) {
+      setInterval(() => this.fetchData(), checkInterval);
+    }
+    this.fetchData();
+  },
+  methods: {
+    fetchData: function () {
+      const handleError = (e) => {
+        console.error(e);
+        this.serverError = true;
+      };
+
+      // Fetch all containers (including stopped) from Docker Socket Proxy
+      fetch(`${this.endpoint}/containers/json?all=true`) // Docker endpoint for container statuses
+        .then((response) => response.json())
```

Comment:
Could you use the [this.fetch](https://github.com/bastienwirtz/homer/blob/main/src/mixins/service.js#L15) method instead ? it handles the json decoding, and applies the global proxy settings.

```suggestion
      this.fetch("/containers/json?all=true") // Docker endpoint for container statuses
```